packages/gestalt/src/icons/compose.svg

```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path fill="#B5B5B5" d="M2.337 19.942l5.671-1.977L19.265 6.706c.981-.98.981-2.57 0-3.55l-1.42-1.421a2.51 2.51 0 0 0-3.55 0L3.036 12.992l-1.978 5.671a1.005 1.005 0 0 0 1.279 1.279M23 22c0 .55-.45 1-1 1H2c-.55 0-1-.45-1-1s.45-1 1-1h20c.55 0 1 .45 1 1"/></svg>
```

Comment:
just want to verify that this svg code went through @jaymarsh and was optimized?

Reply:
I just noticed the `fill` property is wrong here, I know we set colors using the props on the `Icon` component but do you mind updating this SVG source code to stay consistent with the rest of the icon set? You can look at the `pin` icon raw code as an example https://raw.githubusercontent.com/pinterest/gestalt/master/packages/gestalt/src/icons/pin.svg